### PR TITLE
feat: Add Alpine-based image

### PR DESCRIPTION
**Pull request by astencel-sumo**
This is mostly to get rid of the vulnerabilities that the Debian image carries with it.

### CODE REVIEW

File: fluent.conf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    @type file
+    @id   output1
+    path         /fluentd/log/data.*.log
+    symlink_path /fluentd/log/data.log
+    append       true
+    time_slice_format %Y%m%d
+    time_slice_wait   10m
+    time_format       %Y%m%dT%H%M%S%z
+  </match>
+</label>
```
Comment (sumo-drosiek) [inline fluent.conf:33]: Is it from upstream?

Comment (astencel-sumo) [reply fluent.conf:33]: Yes it is, from here: [https://github.com/fluent/fluentd-docker-image/blob/master/v1.12/debian/fluent.conf](https://github.com/fluent/fluentd-docker-image/blob/209628f474f60bc6ff05c8c1ab8ca06dd538c44d/v1.12/debian/fluent.conf)

**Comment by sumo-drosiek** [comment]
Is it ready for review? @astencel-sumo

**Comment by astencel-sumo** [comment]
> Is it ready for review? @astencel-sumo 

yes it is, there's just this one check that fails in GH Actions and passes on my machine 😬

**Comment by astencel-sumo** [comment]
OK the failing test is now fixed with #309. PR is ready.

**Review by sumo-drosiek — APPROVED** [review]
LGTM

**Pull request merged**